install/jakartaee/bin/ts.jte.jdk9

```diff
@@ -1410,12 +1410,11 @@ sigTestClasspath=${s1as.modules}/cdi-api.jar${pathsep}${s1as.modules}/javax.json
 ##   optional.tech.packages.to.ignore=javax.foo.pkg1,javax.bar.pkg2,com.blah.foo.pkg3
 ##
 ##  Additional notes:
-##    - If no packages are to be ignored, use "null". 
-##    - For EE 7 (RI) Full Platform use:
-##           optional.tech.packages.to.ignore=null 
+##    - always ignore the optional JAXB packages: 
+##           optional.tech.packages.to.ignore=jakarta.xml.bind 
 ##  
 #########################################################################
-optional.tech.packages.to.ignore=null
+optional.tech.packages.to.ignore=jakarta.xml.bind
```

Comment:
we need to add "optional.tech.packages.to.ignore=jakarta.xml.bind" at install/jaxrs/bin/ts.jte files.

Reply:
> does web-profile support jaxb? 
No, only Full Platform supports jaxb (although GlassFish Web Profile does include jaxb classes).

if yes javaee_web_profile marker needs to be marked as javaee_web_profile_optional.
> if web-profile doesn't support jaxb, then javaee_web_profile needs to be removed.

Should we remove javaee_web_profile only for JAXB tests (e.g. marked as xml_binding)?

Reply:
can we make  javaee_web_profile as javaee_web_profile_optional for the tests having dependency on xml_binding?

Reply:
> Since this dependency needs to be looked with perspective more of JAXRS and less with web-profile.
> can we make javaee_web_profile as javaee_web_profile_optional for the tests having dependency on xml_binding?

Sure, thanks, I agree.  Will update this.

Reply:
`jaxrs_optional javaee_optional jaxrs_web_profile_optional javaee_web_profile_optional xml_binding`
does ^ marker helps?